mipengine/node/monetdb_interface/common_actions.py

```diff
@@ -168,10 +172,34 @@ def get_table_data(table_name: str) -> List[List[Union[float, int, str, None]]]:
         WHERE tables.system=false
         """
     )
-
+    data = list(zip(*data))
     return data
 
 
+def get_columns_data(
+    schema, data
+) -> List[Union[ColumnDataInt, ColumnDataStr, ColumnDataFloat]]:
```

Comment:
it could return
```
-> List[_ColumnData]:
```
Also what is the difference in purpose between `get_table_data` and `get_columns_data`?

Reply:
get_table_data returns only the data from the database column-stored.
get_columns_data will return a list of _ColumnData which contain for each column the name, type and data.